var/spack/repos/builtin/packages/r-summarizedexperiment/package.py

```diff
@@ -34,16 +34,23 @@ class RSummarizedexperiment(RPackage):
     homepage = "https://bioconductor.org/packages/SummarizedExperiment/"
     git      = "https://git.bioconductor.org/packages/SummarizedExperiment.git"
 
+    version('1.10.0', commit='7ad2e991c8285bfc4b2e15b29d94cc86d07f8f2b')
     version('1.8.1', commit='9d8a29aa9c78bbc7dcc6472537e13fc0d11dc1f7')
     version('1.6.5', commit='ec69cd5cfbccaef148a9f6abdfb3e22e888695d0')
 
-    depends_on('r-genomicranges@1.27.22:', type=('build', 'run'), when='@1.6.5')
-    depends_on('r-genomicranges@1.29.14:', type=('build', 'run'), when='@1.8.1')
+    depends_on('r-genomicranges@1.27.22:', when='@1.6.5', type=('build', 'run'))
+    depends_on('r-genomicranges@1.29.14:', when='@1.8.1', type=('build', 'run'))
+    depends_on('r-genomicranges@1.31.17:', when='@1.10.0', type=('build', 'run'))
     depends_on('r-biobase', type=('build', 'run'))
-    depends_on('r-delayedarray@0.1.9:', type=('build', 'run'), when='@1.6.5')
-    depends_on('r-delayedarray@0.3.20', type=('build', 'run'), when='@1.8.1')
+    depends_on('r-delayedarray@0.1.9:', when='@1.6.5', type=('build', 'run'))
+    depends_on('r-delayedarray@0.3.20:', when='@1.8.1:', type=('build', 'run'))
     depends_on('r-matrix', type=('build', 'run'))
     depends_on('r-s4vectors', type=('build', 'run'))
+    depends_on('r-s4vectors@0.17.25:', when='@1.10.0', type=('build', 'run'))
     depends_on('r-iranges', type=('build', 'run'))
+    depends_on('r-iranges@2.13.16:', when='@1.10.0', type=('build', 'run'))
     depends_on('r-genomeinfodb', type=('build', 'run'))
-    depends_on('r@3.4.0:3.4.9', when='@1.6.5:')
+    depends_on('r-genomeinfodb@1.13.1:', when='@1.10.0', type=('build', 'run'))
+    depends_on('r-biocgenerics@0.15.3:', when='@1.6.5:', type=('build', 'run'))
+    depends_on('r@3.4.0:3.4.9', when='@1.6.5', type=('build', 'run'))
+    depends_on('r@3.5.0:3.5.9', when='@1.10.0', type=('build', 'run'))
```

Comment:
1.8.1 has no r dependency anymore

Reply:
It would be great if you could provide a _range_ of versions of the package that correlate with a _range_ of versions of R. But I realize this can be difficult to track down without downloading every version and seeing when it changes.